src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/android/CommonPage.java

```diff
@@ -32,4 +39,14 @@ public AbstractPage clickBottomNavigatorButton(BottomNavigatorButtons button) {
         navigationButton.format(button.getOptionAndroid()).click();
         return initPage(getDriver(), button.getPage());
     }
+
+    @Override
+    public void clickCloseButton() {
```

Comment:
no idea what this method clickCloseButton does and no idea what's the button closeButton. plz, use appropriate names for methods and variables so it would be understandable